pallets/multisig/src/benchmarking.rs

```diff
@@ -0,0 +1,284 @@
+    }: _(alice.origin, alice_auth_id)
+
+    accept_multisig_signer_as_key {
+        let alice = <UserBuilder<T>>::default().build_with_did("alice", 0);
+        let bob = <UserBuilder<T>>::default().build("bob", 0);
+        let multisig = <MultiSig<T>>::get_next_multisig_address(alice.account());
+        <MultiSig<T>>::create_multisig(alice.origin(), vec![Signatory::Account(bob.account())], 1)?;
+        let auth_id = get_last_auth_id::<T>(&Signatory::Account(bob.account()));
+    }: _(bob.origin, auth_id)
+
+    add_multisig_signer {
+        let alice = <UserBuilder<T>>::default().build_with_did("alice", 0);
+        let bob = <UserBuilder<T>>::default().build("bob", 0);
+        let multisig = <MultiSig<T>>::get_next_multisig_address(alice.account());
+        <MultiSig<T>>::create_multisig(alice.origin(), vec![Signatory::from(alice.did())], 1)?;
+        let origin = RawOrigin::Signed(multisig);
+    }: _(origin, Signatory::Account(bob.account()))
+
+    remove_multisig_signer {
+        let alice = <UserBuilder<T>>::default().build_with_did("alice", 0);
+        let bob = <UserBuilder<T>>::default().build_with_did("bob", 0);
+        let signers = vec![
+            Signatory::from(alice.did()),
+            Signatory::from(bob.did()),
+        ];
+        let multisig = generate_multisig_with_signers::<T>(alice.account(), alice.origin(), signers, 1)?;
+        let origin = RawOrigin::Signed(multisig);
+    }: _(origin, Signatory::from(bob.did()))
+
+    add_multisig_signers_via_creator {
+        // Number of signers
+        let i in 1 .. T::MaxSigners::get();
+
+        let alice = <UserBuilder<T>>::default().build_with_did("alice", 0);
+        let multisig = <MultiSig<T>>::get_next_multisig_address(alice.account());
+        <MultiSig<T>>::create_multisig(alice.origin(), vec![Signatory::from(alice.did())], 1)?;
+        let mut signers = vec![];
+        generate_signers::<T>(&mut signers, i as usize);
+    }: _(alice.origin, multisig, signers)
+
+    remove_multisig_signers_via_creator {
+        // Number of signers
+        let i in 1 .. T::MaxSigners::get();
+
+        let alice = <UserBuilder<T>>::default().build_with_did("alice", 0);
+        let bob = <UserBuilder<T>>::default().build_with_did("bob", 0);
+        let signers = vec![
+            Signatory::from(alice.did()),
+            Signatory::from(bob.did()),
+        ];
+        let multisig = generate_multisig_with_signers::<T>(alice.account(), alice.origin(), signers, 1)?;
+        let signers = vec![
+            Signatory::from(bob.did()),
+        ];
+    }: _(alice.origin, multisig, signers)
+
+    change_sigs_required {
+        let alice = <UserBuilder<T>>::default().build_with_did("alice", 0);
+        let signers = vec![Signatory::from(alice.did())];
+        let multisig = generate_multisig::<T>(alice.account(), alice.origin(), signers)?;
+        let origin = RawOrigin::Signed(multisig);
+    }: _(origin, 1)
```

Comment:
Add two signers initially and then reduce the count to 1. Right now, this is a no-op.